multiphysics/unsteady_fsi_python/readHistoryModal.m

```diff
@@ -0,0 +1,84 @@
+%% \file readHistoryModal.m
+%  \brief Reads the StructHistoryModal.dat file
+%  \authors Nicola Fonzi, Vittorio Cavalieri
+%  \version 7.0.8 "Blackbird"
+%
+% SU2 Project Website: https://su2code.github.io
+%
+% The SU2 Project is maintained by the SU2 Foundation
+% (http://su2foundation.org)
+%
+% Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+%
+% SU2 is free software; you can redistribute it and/or
+% modify it under the terms of the GNU Lesser General Public
+% License as published by the Free Software Foundation; either
+% version 2.1 of the License, or (at your option) any later version.
+%
+% SU2 is distributed in the hope that it will be useful,
+% but WITHOUT ANY WARRANTY; without even the implied warranty of
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+% Lesser General Public License for more details.
+%
+% You should have received a copy of the GNU Lesser General Public
+% License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+
+function [t,q,qdot,qddot] = readHistoryModal(filename,nmodes,n,display)
+

```

Comment:
It would be nice to add some comments what those functions do. The filenames already hint that but imo adding information to in/output of those functions cannot hurt without bloating comments everywhere... and explanation what they are meant for or what a script does (in case it is not a function definition). 
Considering that Matlab is proprietary and one has to assume people dont have it, this becomes especially interesting if they want to recreate the output. I would have preferred python here in the first place (and I assume that these scripts could be written in python and I might be wrong with that) ... BUT it generally is great to add postprocessing scripts like you do here so thumbs up for that 👍 imo that really adds to the value of a tutorial

Reply:
They might run on Octave too.

Reply:
I perfectly agree, Python would have been perfect. Unfortunately, we started working on the postprocessing with Matlab for ease of use and then a lot of postprocessing functions accumulated.
It is definitely in my TODO list to transfer them to python3.
In any case, I do not use complex functions from Matlab so I think, as suggested by @pcarruscag, that they may run in Octave.
I can add some comments to make them more self explaining so that if someone has to modify them, they are easy to read

Reply:
ok cool, I wasn't aware that [The Octave syntax is largely compatible with Matlab](https://www.gnu.org/software/octave/index) ... big-brain-move of octave to snatch some matlab users